options.mk

```diff
@@ -91,3 +91,8 @@ credentials+discovery+ecommerce+insights+lms+registrar+cms
 # Separated by plus signs. Listed in alphabetical order for clarity.
 THIRD_PARTY_SERVICES ?= \
 chrome+coursegraph+elasticsearch710+firefox+memcached+mongo+mysql57+mysql80+opensearch12+redis+namenode+datanode+resourcemanager+nodemanager+sparkmaster+sparkworker+vertica
+
+# Set default theme for MFEs
+ifndef PARAGON_BRAND_PACKAGE
+  PARAGON_BRAND_PACKAGE = edx/brand-edx.org
+endif
```

Comment:
Given MFEs can define the brand package (and its installed version) independently across each MFE, are there any alternative ways of setting the `PARAGON_BRAND_PACKAGE` on a per-MFE basis? 

For example, `frontend-app-learning` might want to use `v3` of the brand package whereas `frontend-app-learner-dashboard` might want `v2`. As is, I believe this would use the latest version of the package, regardless of compatibility with the consuming MFE. This MFE-specific support would largely reflect how brand package MFEs are configured via edx-internal today.

Perhaps each MFE configured via `docker-compose.yml`, extending `microfrontend.yml`, could define it as an environment variable that can be consumed by `microfrontend.yml`?